lib/db.js

```diff
@@ -512,19 +514,30 @@ module.exports = (
       lastAccessTime: token.lastAccessTime,
       createdAt: token.createdAt,
       isMemoryToken: true
-    }]
+    }
     let sessionTokens = []
+    return getGeoData(ip)
```

Comment:
It would be interesting to know if it's about the same on prod-spec instances (whatever prod-spec is). Looking at Kibana, most requests to `/certificate/sign` seem to take around 9 or 10 milliseconds at the moment, so potentially we're trebling that number.

I still think it's worth caching the lookup based on a hash of the IP address fwiw but, as I'm the only one who think so, let's leave it out of this PR. Looking at the rest of the code, I can see a few places where I want to clean up redundant calls to `getGeoData`, so maybe it makes sense to tackle them all together. I'll open a separate issue for that.